lib/beaker/dsl/helpers/puppet_helpers.rb

```diff
@@ -323,7 +323,7 @@ def bounce_service host, service, curl_retries = nil, port = nil
           curl_retries = 120 if curl_retries.nil?
           port = options[:puppetserver_port] if port.nil?
           if host.graceful_restarts?
-            apachectl_path = host.is_pe? ? "#{host['puppetsbindir']}/apache2ctl" : 'apache2ctl'
+            apachectl_path = host.is_pe? ? "#{host['puppetsbindir']}/apachectl" : 'apachectl'
```

Comment:
Opensource puppet tests against passenger packages here but only for the Debian wheezy and trusty cells: https://jenkins.puppetlabs.com/view/legacy_Puppet/view/3.x/job/legacy_puppet_integration-system_linux_3.x/ We'd want to make sure `apachectl` is present on those systems before merging this.

That said I'd be more concerned about module pipelines that might be testing against a larger set of supported Debian platforms and whether `apachectl` is always present?

A safer change might be to test if the `apache2ctl` exists on the host, and if not, fall back to `apachectl`.